docs/UserGuide.md

```diff
@@ -238,10 +255,12 @@ Notify the user if a certain stock is below threshold
 
 | Action    | Format, Examples                                                                                    |
 |-----------|-----------------------------------------------------------------------------------------------------|
-|**Add**    | `add n/NAME q/QUANTITY [s/SUPPLIER] [max/MAX_QUANTITY] [t/TAG]...​` <br> e.g., `add n/Chicken q/3 s/ShengSiong t/Poultry` |
-|**Clear**  | `clear`                                                                                             |
-|**Delete** | `delete INDEX`<br> e.g., `delete 3`                                                                 |
-|**Edit**   | `edit INDEX [n/NAME] [q/QUANTITY] [s/SUPPLIER] [max/MAX_QUANTITY] [t/TAG]…​`<br> e.g.,`edit 1 n/Chicken q/50`                |
-|**Find**   | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find Chicken Steak`                                       |
-|**List**   | `list`                                                                                              |
+|**Add to Inventory**    | `add-i n/NAME q/QUANTITY [s/SUPPLIER] [max/MAX_QUANTITY] [t/TAG]...​` <br> e.g., `add n/Chicken q/3 s/ShengSiong t/Poultry` |
+|**Clear from Inventory**  | `clear-i`                                                                                             |
+|**Delete from Inventory** | `delete-i INDEX`<br> e.g., `delete 3`                                                                 |
+|**Edit Inventory**   | `edit-i INDEX [n/NAME] [q/QUANTITY] [s/SUPPLIER] [max/MAX_QUANTITY] [t/TAG]…​`<br> e.g.,`edit 1 n/Chicken q/50`                |
+|**Find in Inventory**   | `find-i PREFIX KEYWORD [MORE_KEYWORDS]`<br> e.g., `find-i n/Chicken Steak`                                       |
+|**List Inventory**   | `list-i
+|**Remove from Inventory** | `remove-i INDEX q/QUANTITY`                                                                                              |
+|**Find in Deliveries**  | `find-d PREFIX KEYWORD [MORE_KEYWORDS]` <br> e.g., `find-d n/Alex`           |
```

Comment:
Shall we have multiple summary tables for each component (inventory vs delivery), since not all commands necessarily overlap?

Nothing to change here, just need your suggestion. I can make the amendments in my PR.

Reply:
Yeah sounds like a good idea!